src/reducers/moviesReducer.js

```diff
@@ -0,0 +1,9 @@
+export const moviesReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'ADD_MOVIES':
+      return [...action.movies];
```

Comment:
I got a little confused at first by this switch case since there won't be any functionality to add a movie.   There will be functionality to add a ranking though.   Could we consider tweaking the naming if we use this for adding the ranking?